utils/datavzrd/wrapper.py

```
@@ -7,27 +7,30 @@
-        f,
-        outfile=processed,
-        variables={
-            "snakemake": snakemake,
-            # keep the ones below for backwards compatibility
-            "params": snakemake.params,
-            "wildcards": snakemake.wildcards,
-            "input": snakemake.input,
-        },
-        require_use_yte=True,
-    )
+    with open(snakemake.log[0], "a") as log_file:
```

Ruff (0.15.14)

[error] Undefined name `snakemake`

(F821)